pyls/plugins/folding.py

```diff
@@ -0,0 +1,169 @@
+# pylint: disable=len-as-condition
+
+import re
+
+import parso
+import parso.python.tree as tree_nodes
+
+from pyls import hookimpl
+
+SKIP_NODES = (tree_nodes.Module, tree_nodes.IfStmt, tree_nodes.TryStmt)
+IDENTATION_REGEX = re.compile(r'(\s+).+')
+
+
+@hookimpl
+def pyls_folding_range(document):
+    program = str(document.source) + '\n'
```

Comment:
Is `str` really necessary here? I think it'd be better to write

    program = document.source + u'\n'
	
instead.